Quaver.Shared/Database/Maps/MapsetHelper.cs

```diff
@@ -120,10 +120,7 @@ internal static List<Mapset> OrderMapsetsByConfigValue(IEnumerable<Mapset> mapse
         /// </summary>
         /// <param name="mapsets"></param>
         /// <returns></returns>
-        internal static List<Mapset> OrderMapsetsByDateAdded(IEnumerable<Mapset> mapsets)
-        {
-            return mapsets.OrderBy(x => x.Maps[0].DateAdded).ToList();
-        }
+        internal static List<Mapset> OrderMapsetsByDateAdded(IEnumerable<Mapset> mapsets) => mapsets.OrderByDescending(x => x.Maps[0].DateAdded).ToList();
```

Comment:
I've noticed that when you click "Artist", "Title" or "Creator" and then click "Date Added" it'll still sort by the previously selected one. This should be sorted by Date Added -> Artist -> Title

Reply:
Change `x.Maps[0]` to `x.Maps.First()`

Reply:
`internal static List<Mapset> OrderMapsetsByDateAdded(IEnumerable<Mapset> mapsets) => mapsets.OrderByDescending(x => x.Maps.First().DateAdded).ThenBy(x => x.Maps[0].Artist).ThenBy(x => x.Maps[0].Title).ToList();`
Do I still use `x.Maps[0]` after `.ThenBy` or should I change those to `x.Maps.First()` too

Reply:
Wherever you use `[0]` use `.First()`, wherever you use `[length - 1]`, use `.Last()`